chcemvediet/apps/inforequests/templates/admin/inforequests/action/delete_selected_confirmation.html

```diff
@@ -0,0 +1,34 @@
+{% extends "admin/delete_selected_confirmation.html" %}
+{% load amend before after set_attributes from poleno.amend %}
+
+{% block content %}
+  {% amend %}
+    {{ block.super }}
+    {% after path="./ul[last()]" %}
+      {% if outbound %}
+        <p>Outbound messages will be deleted:</p>
+        <ul>{{ outbound|unordered_list }}</ul>
+      {% endif %}
+      {% if inbound %}
+        <p>Inbound messages will be marked undecided:</p>
+        <ul>{{ inbound|unordered_list }}</ul>
+      {% endif %}
+    {% endafter %}
+    {% before path=".//form" %}
+      {% if delete_warnings %}
+        <ul class="messagelist">
+          {% for warning in delete_warnings %}
+            <li class="warning">{{ warning }}</li>
+          {% endfor %}
+        </ul>
+      {% endif %}
+      {% if delete_constraints %}
+        <div class="errornote">
+          <p>Delete not allowed.</p>
+          <ul>{{ delete_constraints|unordered_list }}</ul>
```

Comment:
Preco pri `delete_constraints` mas `delete_constraints|unordered_list` a pri `delete_warnings` to vypisujes po jednom? V principe oba su OK. Ale bolo by fajn vsade pouzivat rovnaky pristup.

Reply:
`django/contrib/admin/static/admin/css/base.css` definuje warning iba v ramci `<ul>`
error je definovany aj ako samostatna classa 
```
ul.messagelist li.warning {
    ...
}

.errornote {
    ...
}

Reply:
:ok